plugins/polio/js/src/domains/VaccineModule/StockManagement/StockVariation/Table/CampaignNameWithWarning.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FunctionComponent } from 'react';
+import { REGULAR, ROUND_ON_HOLD } from '../../constants';
+import { Box, Tooltip } from '@mui/material';
+import { useSafeIntl } from 'bluesquare-components';
+import WarningAmberOutlinedIcon from '@mui/icons-material/WarningAmberOutlined';
+import MESSAGES from '../../messages';
+
+type Props = {
+    text: string;
+    category: string;
+    isVrf?: boolean;
+};
+
+export const CampaignNameWithWarning: FunctionComponent<Props> = ({
+    text,
+    category,
+    isVrf = false,
+}) => {
+    const { formatMessage } = useSafeIntl();
+    if (category === REGULAR) return <span>{text}</span>;
+    const title =
+        isVrf && category === ROUND_ON_HOLD
+            ? formatMessage(MESSAGES.roundsOnHold)
+            : formatMessage(MESSAGES[category]);
+    return (
+        <Box
```

Comment:
<img width="971" alt="Screenshot 2025-06-30 at 11 05 36" src="https://github.com/user-attachments/assets/49bd860f-469e-4a48-9eb0-28c1e7aecbf9" />

Reply:
It's a preexisting error. It comes from the pdf icon button